### PR TITLE
spelling: comments

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

split from #651

### CODE REVIEW

**Comment by jsoref** [comment]
I can't figure out what the failure is talking about. It's why I made #652

**Comment by romani** [comment]
@jsoref , please rebase to fix CI problem.

File: sevntu-checks/src/main/sh/ast.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 #################################################################################
-#  This is a useful script which builds Checkstyle AST (abstract synthax tree)  #
+#  This is a useful script which builds Checkstyle AST (abstract syntax tree)  #
```
Comment (rnveach) [inline sevntu-checks/src/main/sh/ast.sh]: please make the final `#` on this line, line up with the other lines.

Comment (jsoref) [reply sevntu-checks/src/main/sh/ast.sh]: Done

**Pull request merged**